proj/src/main/webapp/static/chat.html

```diff
@@ -50,14 +57,125 @@
+                <div id='submenu3' class="collapse sidebar-submenu">
+                    <a href="#" class="bg-light bg-grey list-group-item list-group-item-action">
+                        <span class="menu-collapsed">edit profile</span>
+                    </a>
+                    <a href="#" class="bg-light bg-grey list-group-item list-group-item-action">
+                        <span class="menu-collapsed">change password</span>
+                    </a>
+                    <a href="#" class="bg-light bg-grey list-group-item list-group-item-action">
+                        <span class="menu-collapsed" id="btnLogout">sign out</span>
+                    </a>
+                </div> 
+                
+                <!-- end of scroll but not the list-->
+                <!--</div>-->
+                <!-- Separator-->
+                <li class="list-group-item sidebar-separator menu-collapsed"></li>
+                <!-- left bottom anchor-->
+                <div class="bottom">
+                    <a href="#top" data-toggle="sidebar-colapse" class="bg-light bg-gold list-group-item list-group-item-action d-flex align-items-center">
+                        <div class="d-flex w-100 justify-content-start align-items-center">
+                            <span id="collapse-icon" class="fa fa-2x mr-3"></span>
+                            <span id="collapse-text" class="menu-collapsed">Collapse</span>
+                        </div>
+                    </a>
+                </div>
+            </ul>
+        </div>
+        <!-- main chat section -->
+        <div class="col p-4">
+            <div class="card" id="main-chat">
+                <h1 class="display-4 card-header" id="big-title"> Chat Room </h1>
+                <!-- chat display-->
+                <div class="card-body" id="message-list">
+                    <!-- template -->
+                    <!-- <ul>
+                        <li class="list-group-item">
+                            <div class="d-flex justify-content-between">
+                                <span>name</span>
+                                <span>time</span>
+                            </div>
+                            <div>message</div>
+                        </li>
+                        <li class="list-group-item">
+                            <div class="d-flex justify-content-between">
+                                <span>name</span>
+                                <span>time</span>
+                            </div>
+                            <div>message</div>
+                        </li>
+                        <li class="list-group-item">
+                            <div class="d-flex justify-content-between">
+                                <span>name</span>
+                                <span>time</span>
+                            </div>
+                            <div>message</div>
+                        </li>
+                    </ul> -->
```

Comment:
Is this not needed? Can it be removed?

Reply:
this is the template for the messages. right now, the messages displayed (at least on my server) only have a message body but not name or time.